Jenkinsfile

```diff
@@ -218,8 +218,9 @@ def run_smoke(targets, toolchains, radioshields, meshinterfaces, raasPort, suite
                 def toolchain = toolchains.keySet().asList().get(j)
                 def radioshield = radioshields.get(k)
                 def meshInterface = meshinterfaces.get(l)
-                // Skip unwanted combination
-                if (target == "NUCLEO_F401RE" && toolchain == "IAR") {
+                // Skip unwanted combination & Wi-SUN compiled only with SL2P shield and vice versa
+                if ((target == "NUCLEO_F401RE" && toolchain == "IAR") || ("${meshInterface}" == "ws" && "${radioshield}" != "S2LP")
```

Comment:
Why Wi-SUN tests are skipped?

Reply:
This follows the build step logic ( https://github.com/ARMmbed/mbed-os-example-mesh-minimal/blob/master/Jenkinsfile#L83 ) where Wi-SUN interface is only built with S2LP and vice versa. This was broken previously as the test step tried to unstash all possible combinations.

Wi-SUN smoke tests are missing from this Jenkinsfile.